test/components/AvroToTypescriptCompiler.ts

```diff
@@ -15,7 +15,7 @@ const avscRoot = `${root}/test/data/avscFieldsData`;
 
 describe("Testing AvroToTypescriptCompiler", () => {
 
-    it(`should create file ./data/tsCompiled/testAvscSchemaUser.ts with same content as in
+    it.skip(`should create file ./data/tsCompiled/testAvscSchemaUser.ts with same content as in
```

Comment:
Is there any reason to skip this test?

Reply:
That was changed by you, I used your branch which already had commits. Should I enable it again?

Reply:
:D

Reply:
Is that a positive